frontend/tests/jest/tests/__snapshots__/ProductListItem.test.tsx.snap

```diff
@@ -21,7 +21,7 @@ exports[`ProductListItem renders and matches the snapshot 1`] = `
           loading="lazy"
           src="https://cart-products.cdn.ifixit.com/cart-products/iPbUhjTn6EYf1PTS.size400?width=320"
           srcset="https://cart-products.cdn.ifixit.com/cart-products/iPbUhjTn6EYf1PTS.size400?width=320 1x, https://cart-products.cdn.ifixit.com/cart-products/iPbUhjTn6EYf1PTS.size400?width=320 2x"
-          style="color: transparent; max-width: 100%; height: auto;"
+          style="color: transparent; object-fit: contain;"
```

Comment:
It's odd skipping a test made this change...

Reply:
Ah, I think that was from the original pull (not from skipping a test). I just updated the snapshot using `pnpm test -- -u` in https://github.com/iFixit/react-commerce/pull/1755/commits/3001956201fdf3a37ca95f7db6be304b40ed86a4 since it was already failing.

Reply:
Ah, I propose we don't base this off of [upgrade-to-next-13](https://github.com/iFixit/react-commerce/tree/upgrade-to-next-13). That way, once this is merged, we can merge in master to `upgrade-to-next-13` and it should pass CI.

Reply:
Nvm, merging this into that PR should be fine too. These commits will just be on that PR as well.

Sorry, I didn't catch this wasn't based off of master before I CRed.

Reply:
Oh, my bad. I just merged it before seeing this message.